ansible_base/authentication/authenticator_plugins/base.py

```diff
@@ -134,3 +135,56 @@ def add_related_fields(self, request, authenticator):
 
     def validate(self, serializer, data):
         return data
+
+    def move_authenticator_user_to(self, new_user, old_authenticator_user):
+        """
+        new_user: django User instance. User that we're moving this account to.
+        old_authenticator_user: AuthenticatorUser instance from this authenticator that is being removed.
+        """
+        exclude_fields = (
+            "social_auth",
+            "authenticator_users",
+            "groups",
+            "has_roles",
+            "role_assignments",
```

Comment:
@relrod I would consider maybe adding `logentry` to this list. Because re-associating the entire activity stream history of a user could time out the request.

I would also consider putting this into a setting, because relationships (similar to logentry, which are toxic in some way or another) will be added downstream.

Reply:
I think `Entry` (not logentry) is fine. Things that inherit `AuditableModel` get an `activity_stream_entries` property, but it's just that - a property, not a real relation. Since activity stream uses GFKs, there's no reverse relation, so I think we're okay?

Reply:
I ran into problems with log entry before. That's actually where this check comes into play:

```python
if not hasattr(old_user, name) or not hasattr(new_user, name):
    continue
```

For some reason, even though `logentry` shows up in fields, it's not actually an attribute on the model.

Either way, I'll add it to the exclude list just in case.